src/index.js

```diff
@@ -86,19 +86,47 @@ function createLogger(options = {}) {
         logger.log(title);
       }
 
-      if (colors.prevState) logger[level](`%c prev state`, `color: ${colors.prevState(prevState)}; font-weight: bold`, prevState);
-      else logger[level](`prev state`, prevState);
+      let prevStateLevel;
```

Comment:
If we want it to be backward compatible, we need to handle different kind of type